modules/publication-service-oaipmh/src/main/java/org/opencastproject/publication/oaipmh/OaiPmhPublicationServiceImpl.java

```diff
@@ -79,453 +80,648 @@
  * Publishes a recording to an OAI-PMH publication repository.
  */
 public class OaiPmhPublicationServiceImpl extends AbstractJobProducer implements OaiPmhPublicationService {
+
   /** Logging facility */
   private static final Logger logger = LoggerFactory.getLogger(OaiPmhPublicationServiceImpl.class);
 
-  /** The publication element mime type */
-  private static final String MIME_TYPE = "text/xml";
+  public static final String SEPARATOR = ";;";
```

Comment:
Using a custom serialization based on the assumption that no items being separate contain this separator does work in practise but is not desirable.

As this issue is not introduced by your work, I've created https://opencast.jira.com/browse/MH-12741 and I'm not expecting you to fix this.

In case your time budget would allow: It would be quite easy to fix this using Gson. Examples:

- Serialize: https://github.com/opencast/opencast/blob/develop/modules/distribution-service-download/src/main/java/org/opencastproject/distribution/download/DownloadDistributionServiceImpl.java#L188
- De-serialize: https://github.com/opencast/opencast/blob/develop/modules/distribution-service-download/src/main/java/org/opencastproject/distribution/download/DownloadDistributionServiceImpl.java#L495